docs/insights-archive-sample/conditional/namespaces/openshift-monitoring/pods/prometheus-operator-5474fcdb86-nknz9/containers/kube-rbac-proxy/logs/last-50-lines.log

```diff
@@ -0,0 +1,6 @@
+2021-12-16T14:24:33.921560406Z I1216 14:24:33.921420       1 main.go:151] Reading config file: /etc/kube-rbac-policy/config.yaml
```

Comment:
Sorry about the logs, it is pretty hard to emulate the issue. That was kind of all that I could get from it.